cli_tools_tests/e2e/variables.go

```diff
@@ -0,0 +1,53 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package e2e
+
+const (
+	projectIDWithoutDefaultServiceAccountFlag                      = "project_id_without_default_service_account"
```

Comment:
Based on the directory of this file, I'm assuming you'll want to re-use the accounts in other test suites. To help clarify your intent, I'd suggest the following changes:

1. Rename the file to `accounts.go` or `projects.go`
2. Create well-named methods that return a struct (as described below). Eg: `ProjectWithDefaultServiceAccount() project`

Reply:
I want to reuse cross test suites, yes. Also, there are more variables to add for rich test scenarios. So I keep variables.go to contain all additional test variables parsing for test suites.